driver-core/src/test/java/com/datastax/driver/core/SpeculativeExecutionTest.java

```diff
@@ -100,14 +102,14 @@ public void should_not_start_speculative_execution_if_first_execution_retries_bu
         // will retry once on this node:
         scassandras.node(1).primingClient().prime(PrimingRequest.queryBuilder()
                         .withQuery("mock query")
-                        .withConsistency(PrimingRequest.Consistency.TWO)
-                        .withResult(PrimingRequest.Result.read_request_timeout)
+                .withConsistency(Consistency.TWO)
+                .withResult(Result.read_request_timeout)
```

Comment:
Ah good catch!  I turned off automatic save actions when switching to Scassandra and must have not turned it back on soon enough, I'll fix all of this and squash.